Java/cp/Reverse_words_in_String.java

```diff
@@ -0,0 +1,33 @@
+/*
+* Given a string, we need to reverse it without reversing any word in it.
+* We can reverse each word of a string.
+* In other words, Reverse text without reversing individual word in a String.
+*/
+    import java.util.Scanner;
+    public class Reverse_String {
+  	public static void main(String[] args) {
```

Comment:
Shift the main logic to some other function and use main() only for the driver code